### PR TITLE
Tailor form user admin endpoint to formId

### PR DESCRIPTION
**Pull request by loneil**
# Description
Original copy paste of the form user endpoint used here was a lot more than needed for Admin FE. FormId wasn't being provided which on large data seemed to be causing DB issues (should only ever call with an ID anyways).
Refactor API service call to simplify it a bunch. Refactor path to match other admin service routes.
Change FE to supply formId in call.

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-258

File: app/tests/unit/routes/v1/admin.spec.js [hunk 1]
```diff
@@ -211,13 +211,13 @@ describe(`GET ${basePath}/users/userId`, () => {
 });
 
 
-describe(`GET ${basePath}/formusers`, () => {
+describe(`GET ${basePath}/forms/formId/formusers`, () => {
```
Comment (jujaga) [inline app/tests/unit/routes/v1/admin.spec.js:214]: formUsers the U should be capitalized. Check other files and locations for case consistency.

**Review by jujaga — APPROVED** [review]
Looks good to me. Just need to make sure everything is consistent to the proper case `formUsers`. Tentatively approving as it appears to fix the bug.

File: app/tests/unit/routes/v1/admin.spec.js (hunk 1 above)
Comment (loneil) [reply app/tests/unit/routes/v1/admin.spec.js]: Fixed. Searched code

**Review by MichaelChung123 — APPROVED** [review]

**Review by TimCsaky — APPROVED** [review]

**Pull request merged**